docs/tutorial.md

```diff
@@ -176,3 +180,53 @@ spec:
           serviceName: frontend
           servicePort: 80
 ```
+
+## Adding Health Probes to your service
+By default, Ingress controller will provision a HTTP GET probe for the exposed pods.  
+The probe properties can customized by adding a Readiness or a Liveness Probe to your deployment/pod spec.
+
+### With `readinessProbe` or `livenessProbe`
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: aspnetapp
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        service: site
+    spec:
+      containers:
+      - name: aspnetapp
+        image: mcr.microsoft.com/dotnet/core/samples:aspnetapp
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 80
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 80
+          periodSeconds: 3
+          timeoutSeconds: 1
+```
+
+*Note:*
+1) `readinessProbe` and `livenessProbe` are supported when configured with `httpGet`.
+2) Probing on a port other than the one exposed on the pod is currently not supported.
+3) `HttpHeaders`, `InitialDelaySeconds`, `SuccessThreshold` are not supported.
+
+###  Without `readinessProbe` or `livenessProbe`
+If the above probes are not provided, then Ingress Controller make an assumption that the service is reachable on `Path` specified for `backend-path-prefix` annotation or the `path` specified in the `ingress` definition for the service.
```

Comment:
Eventually we might want to add a more thorough description of what the implications of the default probe here are.